### PR TITLE
Fix little things in EventPopper

### LINKED ISSUE #1329: Fix things in Eventpopper
- [x] kolla så logiken för vilken färg status dot får stämmer med logiken i lista - använd logiken som används för listitems?
- [x] lägg till event poppers på event overview items
- [x] colors on icons mui style overridden in activitylistitem
- [x] hide checkboxs in event popper
- [x] event popper oveerflowY 'auto" not overflow 'scroll'
- [x] tooltips on all status dots
- [x] "untitled" in multi location thing

### PR DESCRIPTION
**Pull request by ziggabyte**
## Description
This PR addresses a few small/medium things that were not working as expected in the EventPopper. It also deletes two unused old files from the Events components folder.

## Screenshots
bild

## Changes
* Removes the checkboxes, that are to be added when working on bulk operations. 
* Updates logic in useEventClusterData hook to get correct state of an event
* Adds tooltips to the status dots (this is done in a potentially hacky and messy way, as it does not address the general underlying patterns of how we assign colors and acitivity status in slightly different ways from place to place)
* Fixes a logic mistake where "untitled event" was accidentally added in the list of shifts.
* Updates some small styling
* Removes two unused files.

## Related issues
Resolves #1329

### CODE REVIEW

File: src/features/campaigns/components/ActivitiesOverview/items/EventOverviewListItem.tsx [hunk 1]
```diff
@@ -45,6 +48,12 @@ const EventOverviewListItem: FC<EventOverviewListItemProps> = ({
         event.campaign?.id ?? 'standalone'
       }/events/${event.id}`}
       meta={<EventWarningIcons compact model={model} />}
+      onClick={(x: number, y: number) => {
+        openEventPopper(
+          { events: [event], kind: CLUSTER_TYPE.SINGLE },
```
Comment (richardolsson) [inline src/features/campaigns/components/ActivitiesOverview/items/EventOverviewListItem.tsx:53]: Nice!

File: src/features/events/components/EventPopper/SingleEvent.tsx [hunk 2]
```diff
@@ -112,12 +112,12 @@ const SingleEvent: FC<SingleEventProps> = ({ event, onClickAway }) => {
   return (
     <>
       <Box alignItems="center" display="flex">
-        <Checkbox size="medium" />
+        {/*TODO: <Checkbox size="medium" /> */}
```
Comment (richardolsson) [inline src/features/events/components/EventPopper/SingleEvent.tsx:115]: Nice!

File: src/features/events/components/EventPopper/MultiEventPopper/MultiEventListItem.tsx [hunk 3]
```diff
@@ -60,9 +60,10 @@ const MultiEventListItem: FC<MultiEventListItemProps> = ({
                 <LocationName location={event.location} />
               )}
               {clusterType == CLUSTER_TYPE.MULTI_SHIFT && timeSpan}
-              {(clusterType == CLUSTER_TYPE.ARBITRARY &&
-                (event.title || event.activity?.title)) ||
-                messages.common.noTitle()}
+              {clusterType == CLUSTER_TYPE.ARBITRARY &&
```
Comment (richardolsson) [inline src/features/events/components/EventPopper/MultiEventPopper/MultiEventListItem.tsx:63]: Nice catch!

File: src/features/events/components/EventList/index.tsx [hunk 4]
```diff
@@ -1,45 +0,0 @@
-import React from 'react';
```
Comment (richardolsson) [inline src/features/events/components/EventList/index.tsx:1]: Thanks for getting rid of this!

File: src/features/events/utils/getStatusDotLabel.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+import messageIds from '../l10n/messageIds';
+import { Msg } from 'core/i18n';
+import { STATUS_COLORS } from 'features/campaigns/components/ActivityList/items/ActivityListItem';
+
+export default function getStatusDotLabel({
+  color,
+  state,
+}: {
+  color?: STATUS_COLORS;
+  state?: EventState;
```
Comment (richardolsson) [inline src/features/events/utils/getStatusDotLabel.tsx:11]: I don't love this, but I understand the need with the current mess that is status management. We will have to return to this later and clean up, maybe!

**Review by richardolsson — APPROVED** [review]
All good!

File: src/features/events/utils/getStatusDotLabel.tsx (hunk 5 above)
Comment (ziggabyte) [reply src/features/events/utils/getStatusDotLabel.tsx:11]: Yep, it's a mess definitely 🤣

**Pull request merged**